### PR TITLE
Controllers specs

### PR DESCRIPTION
**Pull request by sonyco-4u**
**In this project I did**
Add RSpec as a gem to My project.
Create a Request spec file for all My controllers.
Make sure that for each action you check:
If response status was correct.
If a correct template was rendered.
If the response body includes correct placeholder text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,51 +175,27 @@ Example:
...
-  - [Answer_2]
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./License.md) licensed.
```
Comment (rotimiazeez) [inline README.md:198]: - Your MIT license link is broken. Kindly fix it for a more professional README file 😉.

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @sonyco-4u 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- Tests are implemented and passing ✔️
- No Linter error ✔️ 
- Good PR description 👍🏽 
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (sonyco-4u) [reply README.md:198]: Hi, @rotimiazeez  this is what I have on my readme ( This project is [MIT](./License.md) licensed. )

**Review by rotimiazeez — APPROVED** [review]
Hi @sonyco-4u 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- Tests are implemented and passing ✔️
- No Linter error ✔️
- Good PR description ✔️
- Descriptive README ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**